staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -163,6 +163,17 @@ type StatefulSetPersistentVolumeClaimRetentionPolicy struct {
 	WhenScaled PersistentVolumeClaimRetentionPolicyType `json:"whenScaled,omitempty" protobuf:"bytes,2,opt,name=whenScaled,casttype=PersistentVolumeClaimRetentionPolicyType"`
 }
 
+// StatefulSetOrdinals describes the policy used for replica ordinal assignment
+// in this StatefulSet.
+type StatefulSetOrdinals struct {
+	// Start is the number representing the first index that is used to represent
```

Comment:
Use `start` instead of `Start` since this is API doc for client users (who will see `start` as the field).

You use representing twice in the sentence, I'd like to see a longer sentence.  Suggest moving the "if set" part up to be before "defaults to 0", and in 'defaults to 0" add a clause give the example of the default range (`[0,replicas)`).  Add a short sentence suggesting why a user might change this.  Maybe: "The start index may be used to prefer 1-indexed replica names over the default 0-indexed names, or to orchestrate progressive movement of replicas from one statefulset to another"

Reply:
Thanks for the clarifying suggestions. Updated.